### PR TITLE
Failing test case for issue #201

### PR DESCRIPTION
**Pull request by cmb69**
For easier reproduction – don't merge (yet).

### CODE REVIEW

File: tests/gdimagecopyresampled/Makemodule.am [hunk 1]
```diff
@@ -0,0 +1,9 @@
+if HAVE_LIBPNG
+libgd_test_programs += \
+	gdimagecopyresampled/bug00201
+endif
+
+EXTRA_DIST += \
+	gdimagecopyrotated/CMakeLists.txt \
+	gdimagecopyrotated/bug00201_src.png \
```
Comment (pierrejoye) [inline tests/gdimagecopyresampled/Makemodule.am]: Little copy/paste error :)
Should be gdimagecopyresampled

**Comment by cmb69** [comment]
The following patch shows the behavior of gdImageScale():

````
 tests/gdimagecopyresampled/bug00201.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/tests/gdimagecopyresampled/bug00201.c b/tests/gdimagecopyresampled/bug00201.c
index 0375233..d6e66d3 100644
--- a/tests/gdimagecopyresampled/bug00201.c
+++ b/tests/gdimagecopyresampled/bug00201.c
@@ -54,9 +54,7 @@ int main()
     gdImageCopy(logo, logo_source, 0,0, 0,0, SRC_WIDTH,SRC_HEIGHT);
     gdImageDestroy(logo_source);
     
-    scaled_logo = blank_image(DEST_WIDTH,DEST_HEIGHT);
-    gdImageAlphaBlending(scaled_logo, 0);
-    gdImageCopyResampled(scaled_logo, logo, 0,0, 0,0, 200,105, SRC_WIDTH,SRC_HEIGHT);
+    scaled_logo = gdImageScale(logo, 200,105);
     gdImageDestroy(logo);
     
     layers[0] = background;
````

File: tests/gdimagecopyresampled/bug00201.c [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+int main()
+{
+    gdImagePtr layers[2], background, logo_source, logo, scaled_logo, img;
+    FILE *fp;
+
+    background = blank_image(DEST_WIDTH,DEST_HEIGHT);
+
+    fp = gdTestFileOpen("gdimagecopyresampled/bug00201_src.png");
+    logo_source = gdImageCreateFromPng(fp);
```
Comment (pierrejoye) [inline tests/gdimagecopyresampled/bug00201.c]: please use gdTestFileOpen2 so the directory separator will be handled correctly :)

Comment (cmb69) [inline tests/gdimagecopyresampled/bug00201.c]: Thanks (also for the note above). I'll fix that later, when we have decided what's the expected result of this test case. :-)

**Pull request merged**